elasticsearch/Dockerfile

```diff
@@ -8,18 +8,19 @@ USER 0
 
 ENV HOME=/opt/app-root/src \
   JAVA_VER=1.8.0 \
-  ES_VER=2.3.5 \
-  SG_VER=2.3.5.7 \
-  SG_SSL_VER=2.3.5.15 \
-  ES_CLOUD_K8S_VER=2.3.5 \
-  OSE_ES_VER=2.3.5.7 \
+  ES_VER=2.4.1 \
+  SG_VER=2.4.1.8 \
+  SG_SSL_VER=2.4.1.16 \
+  ES_CLOUD_K8S_VER=2.4.1 \
+  OSE_ES_VER=2.4.1.0 \
   ES_CONF=/usr/share/elasticsearch/config \
   INSTANCE_RAM=512G \
   NODE_QUORUM=1 \
   RECOVER_AFTER_NODES=1 \
   RECOVER_EXPECTED_NODES=1 \
   RECOVER_AFTER_TIME=5m \
-  PLUGIN_LOGLEVEL=INFO
+  PLUGIN_LOGLEVEL=INFO \
+  ES_JAVA_OPTS="-Dmapper.allow_dots_in_name=true"
```

Comment:
Would it be better to setup `mapper.allow_dots_in_name` in standard [`elasticsearch.yml`](https://github.com/openshift/origin-aggregated-logging/blob/master/deployer/conf/elasticsearch.yml) file instead?